src/test/util/exec_test.ts

```diff
@@ -13,13 +13,21 @@
  */
 
 import {assert} from 'chai';
+import * as path from 'path';
 import exec, {checkCommand} from '../../util/exec';
 
 suite('src/util/exec', () => {
   suite('exec()', () => {
     test('returns false if current directory does not exist', async () => {
       const testCwd = process.cwd();
       const result = await exec(testCwd, 'pwd');
+
+      // Handle windows difference on paths
+      const pathPieces = result.stdout.split(/\/([a-z])\//);
+      if (pathPieces.length > 1) {
```

Comment:
pwd = /c/Users/stramel/
process.cwd() = C:\Users\stramel\

resolve(pwd) = C:\c\Users\stramel\

Reply:
Appears to be working on Travis.